pkg/assembler/backends/neo4j/certifyVuln.go

```diff
@@ -401,7 +400,6 @@ func (c *neo4jClient) IngestVulnerability(ctx context.Context, pkg model.PkgInpu
 		sb.WriteString(merge)
 		sb.WriteString(returnValue)
 
-		fmt.Println(sb.String())
```

Comment:
you can remove this from your PR if you want. it's part of PR #552

Reply:
Dropped that specific commit from this branch